package.json

```diff
@@ -4,7 +4,7 @@
   "description": "transparently require() jsx from node",
   "main": "index.js",
   "dependencies": {
-    "react-tools": "^0.13.2"
+    "jstransform": ">=11.0.0"
```

Comment:
jstransform is sort of following semver, though perhaps a bit conservatively by bumping version numbers more aggressively (it's tricky with transforms). Honestly I wouldn't expect us to be shipping many more versions - we're in the process of slowly deprecating it.

> were gated on @petehunt

Hey @petehunt, give other people commit access and publishing rights. Like me and @appsforartists.